tests/generators/test_langchain_serve.py

```diff
@@ -0,0 +1,48 @@
+import os
+import pytest
+import requests_mock
+
+from garak.generators.langchain_serve import LangChainServeLLMGenerator
+
+DEFAULT_GENERATIONS_QTY = 10
+
+
+@pytest.fixture
+def set_env_vars():
+    os.environ['LANGCHAIN_SERVE_URI'] = 'http://127.0.0.1:8000'
+    yield
+    os.environ.pop('LANGCHAIN_SERVE_URI', None)
+
+def test_validate_uri():
+    generator = LangChainServeLLMGenerator(name="test-generator")
+    assert generator._validate_uri('http://127.0.0.1:8000') == True
+    assert generator._validate_uri('bad_uri') == False
```

Comment:
For these I get 

```
========================================================================= FAILURES =========================================================================
____________________________________________________________________ test_validate_uri _____________________________________________________________________

    def test_validate_uri():
>       generator = LangChainServeLLMGenerator(name="test-generator")

tests/generators/test_langchain_serve.py:17: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <garak.generators.langchain_serve.LangChainServeLLMGenerator object at 0x70d588ebf5c0>, name = 'test-generator', generations = 10

    def __init__(self, name, generations=10):
        self.name = name
        self.fullname = f"LangChain Serve LLM {self.name}"
        self.generations = generations
        api_uri = os.getenv("LANGCHAIN_SERVE_URI")
        if not self._validate_uri(api_uri):
>           raise ValueError("Invalid API endpoint URI")
E           ValueError: Invalid API endpoint URI

garak/generators/langchain_serve.py:37: ValueError
```

Any ideas?

Reply:
To tests a method with the `@staticmethod` annotation you should not need to instantiate the class
```suggestion
def test_validate_uri():
    assert LangChainServeLLMGenerator._validate_uri('http://127.0.0.1:8000') == True
    assert LangChainServeLLMGenerator._validate_uri('bad_uri') == False
```

Reply:
better, yep, thanks